packages/vue/src/index.ts

```diff
@@ -227,6 +228,8 @@ const Frontegg: PluginObject<PluginOptions> | any = {
         setStoreKey(this, store);
         this.fronteggAuth = Vue.fronteggAuth;
         this.loginWithRedirect = loginWithRedirect.bind(this);
+        // _entitlements was added for hack purposes - we want the user to pass it to have a reactive part in the computed property, then it will get updated
+        this.getEntitlements = (_entitlements: any, keys: string[]) => fronteggApp.getEntitlements(keys);
```

Comment:
pay attention to the PR description. It's a hack and I don't like it

Reply:
maybe we can make _entitlements type more strict to make it more user friendly?